bin/hsw-cli

```diff
@@ -528,157 +495,174 @@ class CLI {
+      case 'wallets':
+        await this.getWallets();
         break;
       case 'watch':
         await this.importAddress();
         break;
-      case 'key':
-        await this.getKey();
-        break;
-      case 'dump':
-        await this.getWIF();
-        break;
-      case 'lock':
-        await this.lock();
-        break;
-      case 'unlock':
-        await this.unlock();
-        break;
-      case 'resend':
-        await this.resendWallet();
-        break;
-      case 'rescan':
-        await this.rescan();
+      case 'zap':
+        await this.zapWallet();
         break;
       default:
         this.log('Unrecognized command.');
+        this.log('');
         this.log('Commands:');
-        this.log('  $ listen: Listen for events.');
-        this.log('  $ get: View wallet.');
-        this.log('  $ master: View wallet master key.');
-        this.log('  $ shared add [xpubkey]: Add key to wallet.');
-        this.log('  $ shared remove [xpubkey]: Remove key from wallet.');
+        this.log('  $ abandon [hash]: Abandon a transaction.');
+        this.log('  $ account create [account-name]: Create account.');
+        this.log('  $ account get [account-name]: Get account details.');
+        this.log('  $ account list: List account names.');
+        this.log('  $ address [account-name]: Derive new address.');
         this.log('  $ balance: Get wallet balance.');
+        this.log('  $ block [height]: View wallet block.');
+        this.log('  $ blocks: List wallet blocks.');
+        this.log('  $ change [account-name]: Derive new change address.');
+        this.log('  $ coins: View wallet coins.');
+        this.log('  $ dump [address]: Get wallet key WIF by address.');
+        this.log('  $ get: View wallet.');
         this.log('  $ history: View TX history.');
+        this.log('  $ import [wif|hex]: Import private or public key.');
+        this.log('  $ key [address]: Get wallet key by address.');
+        this.log('  $ listen: Listen for events.');
+        this.log('  $ lock: Lock wallet.');
+        this.log('  $ mktx [address] [value]: Create transaction.');
         this.log('  $ pending: View pending TXs.');
-        this.log('  $ coins: View wallet coins.');
-        this.log('  $ account list: List account names.');
-        this.log('  $ account create [account-name]: Create account.');
-        this.log('  $ account get [account-name]: Get account details.');
-        this.log('  $ address: Derive new address.');
-        this.log('  $ change: Derive new change address.');
+        this.log('  $ resendwallet [id]: Resend pending transactions for a single wallet.');
         this.log('  $ retoken: Create new api key.');
         this.log('  $ send [address] [value]: Send transaction.');
-        this.log('  $ mktx [address] [value]: Create transaction.');
+        this.log('  $ shared add [xpubkey]: Add key to wallet.');
+        this.log('  $ shared remove [xpubkey]: Remove key from wallet.');
```

Comment:
`shared list` is missing. Also these require account names and since they affect accounts really (not wallets) I think this should be updated like so:

```suggestion
        this.log('  $ shared add [account-name] [xpubkey]: Add key to account.');
        this.log('  $ shared remove [account-name] [xpubkey]: Remove key from account.');
        this.log('  $ shared list [account-name]: List keys in account.');
```